.github/workflows/ci.yml

```diff
@@ -3,11 +3,7 @@ on:
   push:
     branches:
       - main
-    paths-ignore:
-      - '**.md'
   pull_request:
-    paths-ignore:
-      - '**.md'
```

Comment:
We still want to validate formatting with prettier, so I'm removing those `paths-ignore`s